### PR TITLE
Update documentation/example/test references to 1.6.20

### PR DESCRIPTION
**Pull request by IgnatBeresnev**

### CODE REVIEW

File: integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/TestedVersions.kt [hunk 1]
```diff
@@ -5,7 +5,7 @@ internal object TestedVersions {
     val BASE =
         BuildVersions.permutations(
             gradleVersions = listOf("7.3", "6.9"),
```
Comment (vmishenev) [inline integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/TestedVersions.kt:7]: The [latest](https://gradle.org/releases/) Gradlet version is 7.4.2

**Review by vmishenev — APPROVED** [review]

File: integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/TestedVersions.kt (hunk 1 above)
Comment (IgnatBeresnev) [reply integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/TestedVersions.kt:7]: Indeed, might as well update it and AGP :+1:

File: integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/TestedVersions.kt [hunk 2]
```diff
@@ -4,8 +4,8 @@ internal object TestedVersions {
 
     val BASE =
         BuildVersions.permutations(
-            gradleVersions = listOf("7.3", "6.9"),
-            kotlinVersions = listOf("1.6.10", "1.5.31", "1.4.32"),
+            gradleVersions = listOf("7.4.2", "6.9"),
```
Comment (Goooler) [inline integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/TestedVersions.kt:7]:
```suggestion
            gradleVersions = listOf("7.4.2", "6.9.2"),
```

File: integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/TestedVersions.kt [hunk 3]
```diff
@@ -17,12 +17,12 @@ internal object TestedVersions {
      */
     val ANDROID =
         BuildVersions.permutations(
-            gradleVersions = listOf("7.3", *ifExhaustive("7.0")),
-            kotlinVersions = listOf("1.6.10", "1.5.31", "1.4.32"),
-            androidGradlePluginVersions = listOf("7.0.0")
+            gradleVersions = listOf("7.4.2", *ifExhaustive("7.0")),
+            kotlinVersions = listOf("1.6.20", "1.5.31", "1.4.32"),
+            androidGradlePluginVersions = listOf("7.1.2")
```
Comment (Goooler) [inline integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/TestedVersions.kt:22]:
```suggestion
            androidGradlePluginVersions = listOf("7.1.3")
```

**Pull request merged**